cmd/podman/varlink/io.podman.varlink

```diff
@@ -441,7 +441,7 @@ method ExportContainer(name: string, path: string) -> (tarfile: string)
 
 # GetContainerStats takes the name or ID of a container and returns a single ContainerStats structure which
 # contains attributes like memory and cpu usage.  If the container cannot be found, a
-# [ContainerNotFound](#ContainerNotFound)  error will be returned.
+# [ContainerNotFound](#ContainerNotFound)  error will be returned. If the container is not running, an error will be returned.
```

Comment:
Would be nice to document which error was returned if container is not running.

Reply:
Fixed!